torch/_refs/nn/functional/__init__.py

```diff
@@ -87,6 +88,26 @@ def elu(
     return refs.where(refs.gt(a, 0), a, rhs)
 
 
+@register_decomposition(torch.ops.aten.leaky_relu)
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("a",),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT
+)
+def leaky_relu(a: TensorLikeType, negative_slope: float = 0.01, inplace: bool = False) -> TensorLikeType:
+    """
+    Reference implementation of torch.nn.functional.hardswish
```

Comment:
it's not hardswish though, is it?

Reply:
Oops, originally ported hardswish before running into some inconveniences and choosing leaky_relu instead.